src/concurrency/timestamp_ordering_transaction_manager.cpp

```diff
@@ -1058,8 +1035,8 @@ ResultType TimestampOrderingTransactionManager::AbortTransaction(
 
         if (old_prev.IsNull() == false) {
           auto old_prev_tile_group_header = catalog::Manager::GetInstance()
-                                                .GetTileGroup(old_prev.block)
-                                                ->GetHeader();
+              .GetTileGroup(old_prev.block)
+              ->GetHeader();
           old_prev_tile_group_header->SetNextItemPointer(
               old_prev.offset, ItemPointer(tile_group_id, tuple_slot));
         }
```

Comment:
It seems that all the changes are about formatting. I suggest reverting all these changes so that we can trace the person who really changes the logic of the code.